easybuild/easyconfigs/c/CP2K/CP2K-2023.1-foss-2023b.eb

```diff
@@ -0,0 +1,45 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'CP2K'
+version = '2023.1'
+
+homepage = 'https://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/cp2k/cp2k/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['dff343b4a80c3a79363b805429bdb3320d3e1db48e0ff7d20a3dfd1c946a51ce']
+
+builddependencies = [
+    ('flex', '2.6.4'),
+    ('Bison', '3.8.2'),
+]
+
+dependencies = [
+    ('Libint', '2.7.2', '-lmax-6-cp2k'),
+    ('libxc', '6.2.2'),
+    ('libvori', '220621'),
+    ('FFTW', '3.3.10'),
+    ('PLUMED', '2.9.2'),
+]
+if ARCH == 'x86_64':
+    # LIBXSMM is not supported supported on ARM with GCC 12.2.0 and 12.3.0
+    # see https://www.cp2k.org/dev:compiler_support
+    dependencies += [
+        ('libxsmm', '1.17'),
```

Comment:
The CI is failing because it cannot find an easyconfig for this dependency. Looks like there is no open PR for it either. Are you up for making one (should be quite trivial, just a toolchain version bump for https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/l/libxsmm/libxsmm-1.17-GCC-12.3.0.eb)?